File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-10.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int length = nums.length;
        if (length < 2){
            return length;
        }
        int l = 1;
        for (int r = 1; r < length; r++){
            if (nums[r] != nums[r-1]){
                nums[l] = nums[r];
                l++;
            }
        }
        return l;
    }
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-10.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        numSet = set(nums)
        seq = {}
        for num in nums:
            if num-1 not in numSet:
                seq[num] = 0
                i = 0
                while num + i in numSet:
                    seq[num] += 1
                    i += 1
        return max(seq.values()) if seq else 0